src/components/Terms/Terms.react.js

```diff
@@ -282,8 +282,9 @@ class Terms extends Component {
                 indemnify SUSI and its affiliates, officers, agents, and
                 employees from any claim, suit or action arising from or related
                 to the use of the Services or violation of these terms,
-                including any liability or expense arising from claims, losses ,
-                damages, suits, judgments, litigation costs and attorneys’ fees.
+                including any liability or expense arising from claims, losses,
+                damages, suits, judgments, litigation costs, and attorneys’
```

Comment:
costs and attorneys’

Reply:
It's okay I guess - 

https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/

@AakashMallik can you proofread the whole file to see if we're not missing anything just to be sure? 😄

Reply:
Yeah, it would be nice. I checked it completely. In case I missed something let me know.

Reply:
@anshumanv Sure I'll give it a read ASAP

Reply:
@AakashMallik I hope it is fine?

Reply:
@plxity I'll give it a read at around 22:00. Sorry for the delay but I'm a little caught up in something.